src/templates/en/base_chapter.html

```diff
@@ -1,6 +1,8 @@
 {% extends "base/base_chapter.html" %}
 
 {% block title %}{{ metadata.get('title') }} | {{ year }} | The Web Almanac by HTTP Archive{% endblock %}
+{% block citation_title %}The {{ year }} Web Almanac: {{ metadata.get('title') }} chapter{% endblock %}
```

Comment:
Maybe drop "chapter"?
```suggestion
{% block citation_title %}The {{ year }} Web Almanac: {{ metadata.get('title') }}{% endblock %}
```

Reply:
Done.